ConsoleVirtualGUI.cs

```diff
@@ -0,0 +1,42 @@
+﻿
+namespace BetterProductInventory
+{
+    public class ConsoleVirtualGUI
+    { 
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Welcome to this Mangment System, please choose what you want to do!");
+            while (true)
+            {
+                Console.WriteLine("1.Add a New Product");
+                Console.WriteLine("2.View All Existing Products");
+                Console.WriteLine("3.Edit an Exisiting Product");
+                Console.WriteLine("4.Delete a Product");
+                Console.WriteLine("5.Search For a Product");
+                Console.WriteLine("6.Close");
+                int.TryParse(Console.ReadLine(), out var choice);
```

Comment:
is this the right way to use 'TryParse' ?
check this article (quick overview for it)
https://josipmisko.com/posts/c-sharp-tryparse

Reply:
apparently yes , what seems to be the problem ? 
do you want me to add an if statement to warn the user if he interred anything other than the desired input ?